proxmox/datadog_checks/proxmox/data/conf.yaml.example

```diff
@@ -129,6 +129,22 @@ instances:
     #
     # empty_default_hostname: true
 
+    ## @param resource_filters - list of mappings - optional
```

Comment:
Can you ship out an example filled one for this param? Kinda like what we do here:
https://github.com/DataDog/integrations-core/blob/master/openmetrics/datadog_checks/openmetrics/data/conf.yaml.example#L105-L108

This param might be a bit complex with an example.

Reply:
Good idea, will do!